gamemodes/terrortown/gamemode/shared/sh_printmessage_override.lua

```diff
@@ -0,0 +1,63 @@
+---
+-- @desc Overwrites of the default PrintMessage function.
+-- This overwrite fixes some limitations with the default system (usage of
+-- archaic <a href="https://wiki.facepunch.com/gmod/umsg">umsg</a> system) while
+-- also moving `HUD_PRINTCENTER` to the EPOP system.
+-- @author Mineotopia
+
+if SERVER then
```

Comment:
I don't really like the style. I think this should be either splitted up into two files, or combined better to reduce code/comment cloning. Especially the ply:PrintMessage function could be easily combined like this:

```PrintMessage(type, message, SERVER and self or nil)```, or just ```PrintMessage(type, message, self)```, since the client-side PrintMessage only supports 2 arguments.

Reply:
Even now it looks weird, calling on `CLIENT` a function of 2 params with 3

Reply:
So I should change it back to the first version?

Reply:
No but there should be a better way, this seems to be redundant :/

Reply:
Then please suggest something because I have no better idea.